README.md

```diff
@@ -23,3 +23,8 @@ Goals
 	-	Same performance as just using plain JS/HTML/CSS.
 -	Composability
 	-	Nest components to form your entire user interface, seperate them logically as you would any normal Go package.
+
+Community
+=========
+
+- [Join us](https://gophersinvite.herokuapp.com/) in the `#vecty` channel on Gophers Slack!
```

Comment:
I would change the https://gophersinvite.herokuapp.com link to be over "Gophers Slack" rather than "Join us" text.

You could also linkify the "`#vecty` channel" text with a link to https://gophers.slack.com/messages/vecty/. If you're already logged in to Gophers Slack, you'll end up in the right place. But then make it clear how to get an invite to it if you're not.